### ISSUE TITLE
Some people are stuck in the bed between Prim and BW.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#206]
```
Some people are stuck in the bed between Prim and BW.

See:
http://andors.techby2guys.com/viewtopic.php?f=3&t=856
http://andors.techby2guys.com/viewtopic.php?f=2&t=807

We should add a merchant selling potions for every bed we introduce. The other 
solution would be to add monsters dropping very good potions nearby.
```

Original issue reported on code.google.com by `SamuelPl...@gmail.com` on 9 Jul 2011 at 4:27

**Comment by GoogleCodeExporter** [comment]
```
There could be someone downstairs or outside.
```

Original comment by `SamuelPl...@gmail.com` on 9 Jul 2011 at 4:29

**Comment by GoogleCodeExporter** [comment]
```
A journal fragment was added to the cabin in r228, where a previous adventurer 
describes his (or her) way of getting out of the situation.
```

Original comment by `oskar.wi...@gmail.com` on 29 Apr 2012 at 4:01
- Changed state: **Ready**
- Added labels: **Milestone-0.6.11**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
